docker/compose/docker-compose-nfs.yml

```diff
@@ -0,0 +1,91 @@
+#
```

Comment:
I'm still a bit lost. Are `docker-compose-nfs.yml` and `docker-compose-nfs-mount.yml` reference files or do we expect users to deploy with them? I'm worried that we are fixing the size of the bookie pool, but perhaps there is no way to actually parameterize it and if this is a reference file, then each user will have to configure accordingly.

I'm also not entirely clear how we are going to deploy larger bk clusters. Say we want to deploy a 100 bookie cluster. We should be able to do it separately and simply point pravega to the zk ensemble, right?